src/rust-crypto/rust-crypto.ts

```diff
@@ -94,15 +95,14 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
 
         /** The local user's User ID. */
         private readonly userId: string,
-
```

Comment:
please can you not remove these blank lines? it makes it harder to read the comments imho.